setup.cfg

```diff
@@ -94,7 +94,7 @@ doctest_plus = enabled
 text_file_format = rst
 open_files_ignore = "astropy.log" "/etc/hosts" "*.ttf"
 remote_data_strict = true
-addopts = --doctest-rst
+addopts = --doctest-rst --open-files
```

Comment:
This will not do. It is causing Travis CI to time out, which is not the job I fixed in #10164. That fix was only for the "old dependencies" job but now that you are trying to use this globally, we will have to pin `pytest-openfiles` to 0.4.0 globally too. I am not sure if this is desirable, especially given that @astrofrog is investigating.

Reply:
Ah, that's the issue.  Well, I as planning on taking this out on the final commit for this PR anyway.  The only reason it is there is that your CI tests that actually use `--open-files` don't run if the initial tests have a failure, which this does to show that it is an issue.  So I had to force it.

I'll remove this from the PR now and suffice to say everything seems to be fixed.